v2/pkg/core/inputs/hybrid/hmap.go

```diff
@@ -69,6 +71,10 @@ func (i *Input) Close() {
 func (i *Input) initializeInputSources(options *types.Options) error {
 	// Handle targets flags
 	for _, target := range options.Targets {
+		if isCIDR(target) {
```

Comment:
We can use [`iputil.IsCIDR`](https://github.com/projectdiscovery/iputil/blob/5940e7c42cb040deed0dcda9ed41560f70ce2e16/iputil.go#L74)

Reply:
Updated the function to use `iputil.IsCIDR` as per your suggestion.